cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,360 @@
+					{Name: "events", Type: ArrayColumnType{ColumnTypeString}, Codec: "ZSTD(2)"},
+
+					{Name: "serviceName", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "references", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+
+					// custom columns
+					{Name: "responseStatusCode", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "externalHttpUrl", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "httpUrl", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "externalHttpMethod", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "httpMethod", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "httpHost", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "dbName", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "dbOperation", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "hasError", Type: ColumnTypeBool, Codec: "T64, ZSTD(1)"},
+					{Name: "isRemote", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+
+					// attribute cols
+					{Name: "httpRoute", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "msgSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "msgOperation", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "dbSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "rpcSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "rpcService", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "rpcMethod", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "peerService", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+
+					// alias for attribute cols
+					{Name: "attribute_string_http$$route", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "httpRoute"},
+					{Name: "attribute_string_messaging$$system", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "msgSystem"},
+					{Name: "attribute_string_messaging$$operation", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "msgOperation"},
+					{Name: "attribute_string_db$$system", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "dbSystem"},
+					{Name: "attribute_string_rpc$$system", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcSystem"},
+					{Name: "attribute_string_rpc$$service", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcService"},
+					{Name: "attribute_string_rpc$$method", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcMethod"},
+					{Name: "attribute_string_peer$$service", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "peerService"},
```

Comment:
serviceName has no alias.

Reply:
not required for service.name as it is a resource attribute and `service.name`/`serviceName` filter will go to resource attribute. You will see that in further QS pr's

Reply:
I don't get it. Why can't the same be said about other cols then?

Reply:
Ahh, for the group by, it will be required, for filtering it isn't because it directly goes to the resource table.

let me add it.